dataviz/styles/style.css

```diff
@@ -23,7 +24,6 @@ body,
 html {
   position: relative;
   font-family: Helvetica, Arial, sans-serif;
-  overflow: hidden;
```

Comment:
@DianaPerkinsDesign This was the only way I could get the app not to scroll vertically in Safari (works fine in Chrome). Was this rule affecting other styles?

Reply:
Ah okay, if that’s a bug fix then we'll put it back! We didn't want the graph to become inaccessible when the diagram animation is showing.